pkg/util/mount/mount_linux.go

```diff
@@ -68,15 +68,13 @@ func (mounter *Mounter) Mount(source string, target string, fstype string, optio
 	mounterPath := ""
 	bind, bindRemountOpts := isBind(options)
 	if bind {
-		err := doMount(mounterPath, defaultMountCommand, source, target, fstype, []string{"bind"})
-		if err != nil {
+		if err := doMount(mounterPath, defaultMountCommand, source, target, fstype, []string{"bind"}); err != nil {
```

Comment:
I think in the first place, the purpose of moving the err to a separate line was because this line is long as is.